snowplow-tracker/src/main/java/com/snowplowanalytics/snowplow/tracker/tracker/ProcessObserver.java

```diff
@@ -137,7 +137,7 @@ public static void onEnterBackground() {
                     }
                 }
             } catch (Exception e) {
-                Logger.e(TAG, e.getMessage());
+                Logger.e(TAG, "Method onEnterBackground raised an exception: %s", e);
```

Comment:
I'm just picking on this `Logger.e` as the message has changed in this PR but should we change all `Logger.e` to `Logger.track`?

Reply:
As anticipated (sadly, weeks ago) it's not possible at the moment because the tracker doesn't allow service event and it can cause of infinite loops.
For now it's better limiting the error reporting for just few of them.